tests/performance/tools/src/main/java/org/glassfish/jersey/tests/performance/tools/TestDataGeneratorApp.java

```diff
@@ -134,7 +133,7 @@ public static void generateFile(final String resourceRelativeUrl, final int mini
 
         final File file = new File(fileName);
         final BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(
-                new FileOutputStream(file), Charset.forName("UTF-8")));
+                Files.newOutputStream(file.toPath()), Charset.forName("UTF-8")));
```

Comment:
You need to add import for Files:
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.9.0:compile (default-compile) on project performance-test-tools: Compilation failure
/home/jenkins/agent/workspace/ci-cd-integration_PR-5350/tests/performance/tools/src/main/java/org/glassfish/jersey/tests/performance/tools/TestDataGeneratorApp.java:[136,17] cannot find symbol
  symbol:   variable Files
  location: class org.glassfish.jersey.tests.performance.tools.TestDataGeneratorApp

Reply:
Fixed in https://github.com/eclipse-ee4j/jersey/pull/5350/commits/0803e1454860b0cde0b6075aac61108b279b7b8d.

Reply:
(The reason why I could not detect that earlier is that I cannot build from scratch: `mvn compile test-compile` on a fresh clone fails with lots of compilation failures outside of my actual changes).